packages/playwright/src/reporter.ts

```diff
@@ -339,7 +339,7 @@ class ReplayPlaywrightReporter implements Reporter {
         {
           ...testMetadata,
           approximateDuration: test.results.reduce((acc, r) => acc + r.duration, 0),
-          result: status === "interrupted" ? "unknown" : status,
+          result: status,
```

Comment:
According to the playwright types here, the `'interrupted'` status doesn't exist and this prevents this package from compiling successfully